ktor-shared/ktor-serialization/ktor-serialization-kotlinx/ktor-serialization-kotlinx-xml/jvm/test/XmlClientKotlinxSerializationTest.kt

```diff
@@ -39,4 +39,9 @@ class XmlClientKotlinxSerializationTest : AbstractClientContentNegotiationTest()
     @Ignore
     override fun testSerializeNull() {
     }
+
+    @Test
+    @Ignore
+    override fun testSerializeFailureHasOriginalCauseMessage() {
```

Comment:
Is it okay that these tests are ignored? Some of them also have error messages, but they have another format

Reply:
Gson doesn't throw exceptions because it has no handling for non nullable properties
Protobuf throws not helpful exception, like `Expected wire type 2, but found 0`
XML throws an exception before, on type mismatch